app/src/main/java/com/example/geoquiz/MainActivity.kt

```diff
@@ -5,30 +5,69 @@ import android.os.Bundle
 import android.util.Log
 import android.view.Gravity
 import android.widget.Button
+import android.widget.ImageButton
 import android.widget.TextView
 import android.widget.Toast
+import androidx.lifecycle.ViewModelProviders
 
 class MainActivity : AppCompatActivity() {
     private lateinit var trueButton: Button
     private lateinit var falseButton: Button
+    private lateinit var nextButton: ImageButton
+    private lateinit var questionTextView: TextView
+
+    private val quizViewModel by lazy {
+        ViewModelProviders.of(this).get(QuizViewModel::class.java)
+    }
+    private val KEY_INDEX = "index"
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        val currentIndex = savedInstanceState?.getInt(KEY_INDEX, 0) ?: 0
+        quizViewModel.currentIndex = currentIndex
+
         trueButton = findViewById(R.id.true_button)
         falseButton = findViewById(R.id.false_button)
+        nextButton = findViewById(R.id.next_button)
+        questionTextView = findViewById(R.id.question_text_view)
+
 
         trueButton.setOnClickListener {
-            // CHALLENGE: Customize the toast to show at the top instead of the bottom of the screen
-            val truePopUp = Toast.makeText(this,R.string.correct_toast,Toast.LENGTH_LONG)
-            truePopUp.setGravity(Gravity.TOP,0,0)
-            truePopUp.show()
+            checkAnswer(true)
         }
+
         falseButton.setOnClickListener {
-            //CHALLENGE *
-            val falsePopUp = Toast.makeText(this,R.string.incorrect_toast,Toast.LENGTH_LONG)
-            falsePopUp.setGravity(Gravity.TOP,0,0)
-            falsePopUp.show()
+            checkAnswer(false)
+        }
+
+        nextButton.setOnClickListener {
+            quizViewModel.moveToNext()
+            updateQuestion()
         }
+
+        updateQuestion()
+    }
+
+    private fun updateQuestion() {
+        val questionTextResId = quizViewModel.currentQuestionText
+        questionTextView.setText(questionTextResId)
```

Comment:
This can also be written as  `questionTextView.text = getString(questionTextResId)`. With Kotlin, property accessors tend to be used more frequently. This is just a flavor thing, nothing wrong with the way you did it ;)